WindowsUpdates/WindowsUpdates.psm1

```diff
@@ -84,6 +84,8 @@ function Add-WindowsUpdateToCollection {
     $Collection.Add($Update) | Out-Null
 }
 
+$ignoreUpdateDriverHardwareIDs = "{084f01fa-e634-4d77-83ee-074817c03581}","{0f4130dd-19c7-7ab6-99a1-980f03b2ee4e}"
```

Comment:
we shouldn't hardcode the hardware ids, as this might not be the case in other case

Reply:
add a new parameter to Get-WindowsUpdate:
         [Parameter(Mandatory=$false)]
         [array]$ExcludeHardwareId=@()

that can then be used by an external call